lib/redis_client.py

```diff
@@ -0,0 +1,40 @@
+import os
+import redis
+import datetime
+from nypl_py_utils.functions.log_helper import create_log
+
+
+class RedisClient:
```

Comment:
If Redis is commonly used we could add a generalized version of this class to nypl-py-utils